setup.py

```diff
@@ -27,11 +27,11 @@
     description="",
     long_description=README,
     long_description_content_type="text/markdown",
-    url="",
-    author="",
-    author_email="",
-    # maintainer=
-    # maintainer_email=
+    url="https://project.url/",
```

Comment:
You might want to update the README file now that these are no longer "" (empty) to call out to the engineers to change them after "forking" or making a new repo.

Reply:
Oh yes, good point!

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/48/commits/680f985144638c12c37ddeca08cba9be13c52375.